backend/src/app/features/components_v2.clj

```diff
@@ -225,8 +225,8 @@
                     [shape]
                     (if (ctk/instance-head? shape)
```

Comment:
When I wrote this, i forgot that in v1 component roots can be frames, too.

It would be better to apply this assoc only if is an instance head and the type is not already a frame.